.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+-   repo: local
+    hooks:
+    -   id: rspec_tests
+        name: rspec tests˜
+        entry: ./pre-commit.sh˜
```

Comment:
are the `~`s typos?
also, rspec_tests are too specific.. the pre-commit will include linters, acceptance tests and other checks later on, so we probably should stay generic in naming stuff here

Reply:
They where typos from the examples, the tilde's have been removed.
Have renamed rspec tests to just test.